packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -7,7 +7,7 @@ import { FocusZone, FocusZoneDirection, FocusZoneTabbableElements } from '@fluen
 import { IAccessibilityProps, ChartHoverCard, ILegend, Legends } from '../../index';
 import { Pie } from './Pie/index';
 import { IChartDataPoint, IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './index';
-import { getAccessibleDataObject, getColorFromToken, getNextColor } from '../../utilities/index';
+import { getAccessibleDataObject, getColorFromToken, getNextColor, getNextGradient } from '../../utilities/index';
```

Comment:
Add tests for your scenarios.
See details about creating and executing tests here.
https://microsoft.github.io/fluentui-charting-contrib/docs/Testing%20Strategy#:~:text=Manual%20testing-,Test%20Completion,-%3A

Reply:
Added unit tests for `getNextGradient()` and `getGradientFromToken()` in `src\utilities\UtilityUnitTests.test.ts`